api-reference/openapi.yaml

```diff
@@ -90,6 +90,38 @@ components:
               title: Threshold
               description: |
                 A number between 0 and 1 indicating the threshold for document segmentation. Defaults to auto, which uses an automatic threshold.
+            chunking_options:
+              type: object
+              properties:
+                merging_strategy:
+                  type: string
+                  enum: ['header_augmenter', 'greedy_text_element', 'marked']
+                  title: Strategy
+                  description: |
+                    The strategy to use for merging chunks. Defaults to header_augmenter.
+                tokenizer:
+                  type: string
+                  enum: ['openai_tokenizer', 'character_tokenizer', 'huggingface_tokenizer']
+                  title: Tokenizer
+                  description: |
+                    The tokenizer to use for chunking. Defaults to openai_tokenizer.
+                tokenizer_options:
+                  type: object
+                  description: |
+                    The options for the tokenizer. For openai_tokenizer, the options include model_name, which can be set to text-embedding-3-small.
```

Comment:
The second line is confusing. I'd rather remove it, and redirect users to the main documentation page so they can understand it properly.

Reply:
I would rather make the description more descriptive and have it be the same as the processing_options page, but I've now changed it to a link as per your request.